packages/core/docs/guide/checkout.md

```diff
@@ -0,0 +1,145 @@
+# Checkout
+Checkout is a process of providing information about shipping and billing details, and picking certain shipping and payment method. Then with all the data, we are able to place an order and pay for it.
+
+## Shipping
+To access shipping details, we can use property of `useShipping` called `shipping`.
+```js{8,16}
+import { useShipping } from '{INTEGRATION}';
+import { onSSR } from '@vue-storefront/core';
+
+export default {
+  setup () {
+    const {
+      load,
+      shipping
+    } = useShipping();
+
+    onSSR(async () => {
+      await load();
+    });
+
+    return {
+      shipping
+    };
+  }
+}
+```
+`shipping` property will return `null` if the address is not saved on the server or just has not been loaded. In order to save shipping details, you could use `save` method:
```

Comment:
```suggestion
`shipping` property will return `null` if the address is not saved on the server or just has not been loaded. In order to save shipping details, you can use `save` method:
```

also it's worth explaining what `save` means a bit further (eg it will be persisted in a composable state / available each time you call `load` or similar )